addons/stock/models/stock_move_line.py

```diff
@@ -273,6 +273,14 @@ def write(self, vals):
                     if new_product_qty != ml.product_qty:
                         new_product_uom_qty = ml.product_id.uom_id._compute_quantity(new_product_qty, ml.product_uom_id, rounding_method='HALF-UP')
                         ml.with_context(bypass_reservation_update=True).product_uom_qty = new_product_uom_qty
+                        for move in ml.mapped('move_id'):
```

Comment:
`for move in ml.mapped('move_id').filtered(lambda x: x.state == 'assigned'):`
remove on tab :-)